std_msgs/Float32MultiArray states_opti
float32 time_opti